src/main/java/org/spongepowered/common/mixin/core/entity/LivingEntityMixin.java

```diff
@@ -385,7 +388,7 @@ public boolean attackEntityFrom(final DamageSource source, final float amount) {
             return false;
         }
         // Sponge end
-        if (this.isEntityInvulnerable(source)) {
+        if (this.isInvulnerableTo(source)) {
```

Comment:
Should be `shadow$` prefixed if it's calling a shadow method (Yes, I know this is an overwrite).